package.json

```diff
@@ -3,7 +3,11 @@
   "description": "Hoodie Client Core",
   "main": "index.js",
   "scripts": {
-    "test": "standard && npm run -s test:node | tap-spec",
+    "build": "npm-run-all -p build:*",
+    "build:dev": "browserify -s Hoodie index.js -o hoodie.js",
+    "build:min": "browserify -s Hoodie -g uglifyify index.js | uglifyjs -c -o hoodie.min.js",
```

Comment:
In the other repos, we bundle into `/dist` folder. Shall we do that here as well to follow the convention? See https://github.com/hoodiehq/hoodie-client-store/blob/master/package.json#L7 for example